### ISSUE TITLE
bug: _leadingWhiteSpace can't handle no space first line

### ISSUE THREAD

**Issue by huang12zheng** [domain-centric/documentation_builder#93]
* show from source:
```dart
final _leadingWhiteSpace =
    FluentRegex().startOfLine().whiteSpace(Quantity.oneOrMoreTimes());
```
* example:
```dart
print(FluentRegex()
            .multiline(false)
            .startOfLine()
            .literal('c')
            .removeFirst('babab\ncbababa'));
// babab
// cbababa
```

```dart
print(FluentRegex()
            .startOfLine()
            .literal('c')
            .removeFirst('babab\ncbababa'));
// babab
// bababa
```

* suggestion:
add .multiline(false)

**Comment by domain-centric** [comment]
Not sure I understand the problem: 

expect(_leadingWhiteSpace.removeAll('\n   \t  \n hello'),'hello');

seems to work as expected: it should remove all leading white space characters like spaces, tabs and new line characters

**Comment by domain-centric** [comment]
It was meant to remove the leading empty lines, not the leading white spaces of every line...

**Comment by domain-centric** [comment]
Thanks for pointing it out @huang12zheng! Should be fixed in 0.0.3
